erpnext/selling/doctype/sales_order_item/sales_order_item.py

```diff
@@ -8,8 +8,8 @@
 from erpnext.controllers.print_settings import print_settings_for_item_table
 
 class SalesOrderItem(Document):
-	def __setup__(self):
-		print_settings_for_item_table(self)
+    def __setup__(self):
+        print_settings_for_item_table(self)
 
 def on_doctype_update():
-	frappe.db.add_index("Sales Order Item", ["item_code", "warehouse"])
\ No newline at end of file
+    frappe.db.add_index("Sales Order Item", ["item_code", "warehouse"])
```

Comment:
Revert the file changes

Reply:
@hrwX those changes in earlier files were due to pep257- Convert indentation to spaces.
The updated file reviewed has the reverted changes.